pkg/sources/git/git_test.go

```diff
@@ -171,6 +172,16 @@ func Test_generateLink(t *testing.T) {
 			},
 			want: "https://github.com/trufflesec-julian/confluence-go-api/blob/047b4a2ba42fc5b6c0bd535c5307434a666db5ec/.gitignore",
```

Comment:
i believe this test case will fail due to the `L#0` note I mentioned above. If you feel like keeping the L#0 we would just need to update this test as well so it passes.